tools/symbols_map_generator/main.py

```diff
@@ -15,16 +15,31 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import logging
+import sys
+import os
 import argparse
 from pathlib import Path
 from typing import TypedDict, Optional
 from collections import OrderedDict
 import bisect
-import clang.cindex
-import os
 
+logging.basicConfig(level=logging.INFO)
 _logger = logging.getLogger(__name__)
 
+library_not_found_error_msg: str = """\
+    Please install libclang-19-dev and python3-clang-19.
+    Also, ensure that the required environment variables are set correctly. (e.g.)
+        export MIR_SYMBOLS_MAP_GENERATOR_CLANG_SO_PATH=/usr/lib/llvm-19/lib/libclang.so.1
+        export MIR_SYMBOLS_MAP_GENERATOR_CLANG_LIBRARY_PATH=/usr/lib/llvm-19/lib"""
```

Comment:
`libclang-18-dev` is also a viable option and is in `noble/universe`, unlike `libclang-19-dev`.

And, in the future, we'll likely move on to newer versions of clang. So would it be better to specify `libclang-dev` (as the README did) and `python3-clang`?